### ISSUE TITLE
[FR] Allowing to hide the state of the first entity

### ISSUE THREAD

**Issue by Nickduino** [kalkih/mini-graph-card#1150]
Currently, the state of the first entity (whether it's the only one or there are several) cannot be hidden, it ignores the `show_state: false` parameter. Letting the user do this would allow us to declutter some dashboards when we want to look at the shape (and dynamic color) of a graph without the need for a precise value.

Here's a mock-up of the expected result:

![image](https://github.com/user-attachments/assets/eade3309-f6fc-402a-ac3b-e7490fa94c14)

**Comment by ildar170975** [comment]
Same picture may be achieved by
```
show:
  state: false
```

**Comment by ildar170975** [comment]
The problem appears only with several entities in a card. When a card contains only one entity - a state can be hidden easily.

**Comment by Nickduino** [comment]
> The problem appears only with several entities in a card.

That's true. Which makes the current behavior of `show_state: false` even more incomprehensible

**Comment by ildar170975** [comment]
> Which makes the current behavior of show_state: false even more incomprehensible

Imho it can harm ONLY in case of several entities. In case of one entity - a “state” label can be hidden as it was shown.

**Comment by ildar170975** [comment]
Actually, if you do not to show a state for the 1st entity - just make this entity 2nd. Means - there is a simple workaround.

**Comment by Nickduino** [comment]
> Actually, if you do not to show a state for the 1st entity - just make this entity 2nd. Means - there is a simple workaround.

1. It's still just a workaround and not the behavior a user expects
2. You cannot order the graphs the way you want, so you can't always change the order of the entities as seen in https://github.com/kalkih/mini-graph-card/issues/1147

**Comment by KompressR** [comment]
So I have a use case that I think this feature would make possible. I am trying to show my holdings in a stock, my plan was to use the trending attribute to indicate whether the stock is rising or falling by changing the icon color. Since it seems the icon color can only change with the primary entity, I figured I could just hide the state, have my stock value show up first as the secondary entity, number of shares third, and the icon just changing color if its rising or falling. I've already bypassed the color threshold only working on numbers by creating a template number that will set it to 0 on down and 1 on up, and can change the icon no problem, but I still have no way to just change the icon and not show the state.

**Comment by ildar170975** [comment]
@KompressR 
Please post a simplified code of the card which lack the ability to "hide the state of the first entity".

**Comment by Cybaen13** [comment]
Looking for the same thing here,
Seems that the state display is overriden by this condition in **main.js** (_line 287_)
```
renderState(id) {
    const isPrimary = id === 0; // rendering main state element?
    if (isPrimary || this.config.entities[id].show_state) {
      const state = this.getEntityState(id);
```

I believe that replacing `||` by `&&` shall fix the issue as show_state is currently overriden by "isPrimary"
